frontend/src/metabase/query_builder/components/filters/modals/BulkFilterItem/BulkFilterItem.tsx

```diff
@@ -30,16 +32,14 @@ export const BulkFilterItem = ({
   const fieldType = useMemo(() => {
     const field = dimension.field();
 
-    if (BASE_FIELD_FILTERS.includes(field.base_type ?? "")) {
-      return field.base_type;
-    }
-
-    if (field.has_field_values === "list") {
-      return "type/Category";
-    }
+    const relevantFieldType = FIELD_TYPE_PRIORITY.find(t =>
```

Comment:
nit: 
```suggestion
    const relevantFieldType = FIELD_TYPE_PRIORITY.find(type =>
```

Which kind of exposes the code smell a bit as `"list"` isn't really a type but a value of `has_field_values`, and it's in `FIELD_TYPE_PRIORITY`.

To be clear, I'm not sure how to deal with this, I just want to point it out.

Reply:
Good point. I think it can be improved with a better variable name.